src/modules/videoconference/VideoConferenceModule/Toolbar.h

```diff
@@ -54,7 +54,7 @@ class Toolbar
 
     std::wstring theme = L"system";
 
-    bool HideToolbarWhenUnmuted = true;
+    std::wstring ToolbarHide = L"Never";
```

Comment:
```suggestion
    std::wstring ToolbarHide = L"When both camera and microphone are unmuted";
```

Reply:
I think we should leave the "Never" values as they represent how the feature used to work. Automatically selecting something else might confuse users. Also, since this is the way it works by default, it makes sense to keep the default behaviour here even if this does not actually work as default

Reply:
Currently we have `HideToolbarWhenUnmuted` with a default value set to `true`. Now we have 3 possible values for `ToolbarHide`:
- `Never`
- `When both are unmuted`
- `When both are muted` (new feature, so won't be inspected here)

Let's compare current and new behavior:

Current:
- `true` (default): 
    - camera not in use:
        Toolbar stays on screen as long as a mic is muted. It flickers for a short time if you try to mute camera.
    - camera in use:
        Toolbar stays on screen as long as a mic/camera is muted. Everything is unmuted when camera usage session is ended.
- `false`:
    - camera not in use:
        Same as `true`
    - camera in use:
        Toolbar always stays on screen. Everything is unmuted when camera usage session is ended, thus toolbar disappears.

New:
- `Never`:
    - camera not in use:
        Toolbar always stays on screen.
    - camera in use:
        Toolbar always stays on screen.
- `When both are unmuted`:
        - camera not in use:
            same as current behavior with `true`
        - camera in use:
            same as current behavior with `true`, but you've removed the logic to `unmuteAll` when session is ended, so the toolbar will stay on screen even when you end the camera session and unmute mic. That's a bug.

So `Never` introduces a new behavior which is unexpected for user.